core/evancore.vue.libs/src/components/forms/files/files.ts

```diff
@@ -71,8 +71,8 @@ export default class FilesInputComponent extends mixins(ControlComponent) {
    * Empty text that shown, when no files are uploaded and the component is disabled.
    */
   @Prop({
-    default: [ ]
-  }) value: Array<any>;
+    default: [],
+  }) value: UIContainerFile[] | { files: UIContainerFile[] };
```

Comment:
Imo the Prop interface should be just `UIContainerFile[]` and the parent should handle passing the right format. This would also make working with it way easier in this component (see line 97).
Or would this be too much refactoring?